### PR TITLE
Task/3484/email refactoring

### PR DESCRIPTION
**Pull request by pcoder**
This PR proposes email refactoring. These are the 4 emails that have been refactored. 
1. Data Center Light Account Activation
2. Password Reset
3. Your New VM [name] at Data Center Light
4. Virtual Machine Cancellation

I created to base templates for html and plain text emails. All mails extend these templates.
1. datacenterlight/templates/datacenterlight/emails/base_email_datacenterlight.html
2. datacenterlight/templates/datacenterlight/emails/base_email_datacenterlight.txt

Some additional notes
1. Apart from these, I think we may be missing a "Welcome User" email that is sent after a successful registration and account validation.
2. I currently put the "My VM Page" link in Virtual Machine Cancellation template to hosting:orders. To point it to the correct order page, we need task 3780 [We need VM details to be stored in local postgres database] completed because without this we need to fetch VM data from OpenNebula which in turn can not provide this information as this VM has been terminated. So, the user ends up seeing "This VM is not available" message.

### CODE REVIEW

File: datacenterlight/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -102,6 +102,9 @@ msgstr ""
 msgid "SUBMIT"
 msgstr "ABSENDEN"
 
+msgid "Your Data Center Light Team"
+msgstr "Dein Data Center Light Team"
+
 msgid "Thank you for your request."
 msgstr "Vielen Dank für Ihre Anfrage."
```
Comment (verysanghee) [inline datacenterlight/locale/de/LC_MESSAGES/django.po]: Ihre -> Deine

**Comment by verysanghee** [comment]
While testing the mails I realised some part of our hosting app has evolved beyond the mail contents . For example, for VM cancellation mail 
screen shot 2017-09-05 at 10 50 14
a user is told to click "my VM page" to see the order status, but by doing so the user lands in "My Bills", which does not show detail of deleted VMs.
This issue might be partly solved by this ticket, https://redmine.ungleich.ch/issues/3764
and in the meantime we might need to change the btn text and contents slightly. I will assign the modification along with the style fix as a separate task.

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -195,20 +195,101 @@ msgstr "Support / Kontakt"
...
+"Bitte folge diesem Link und wähle ein neues Passwort: %(base_url)s"
+"%(password_reset_url)s Solltest du kein neues Passwort angefordert haben, "
+"dann ignoriere diese E-Mail.\n"
+"Dankeschön!\n"
 
 #, python-format
-msgid "The %(site_name)s team"
+msgid ""
+"You're receiving this mail because your virtual machine [%(vm_name)s] has "
+"been cancelled.<br/>\n"
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: I received vm name as [public-CentOS 7-17582] I think "public" part is not supposed to be there?

**Comment by verysanghee** [comment]
Just realising that i did not read your comment above thoroughly- I ended up writing the things you already laid out well. Sorry :)

**Comment by verysanghee** [comment]
Could you resolve the conflicts? :D

File: hosting/locale/de/LC_MESSAGES/django.po (hunk 2 above)
Comment (pcoder) [reply hosting/locale/de/LC_MESSAGES/django.po]: Good catch @verysanghee I fixed that in the latest commit.

**Comment by verysanghee** [comment]
looks good!

**Pull request merged**